src/Transport/GrpcTransport.php

```diff
@@ -143,8 +153,9 @@ public static function build(string $apiEndpoint, array $config = [])
                 "instead got: " . print_r($channel, true)
             );
         }
+        $logger = isset($config['logger']) ? $config['logger'] : null;
         try {
-            return new GrpcTransport($host, $stubOpts, $channel, $config['interceptors']);
+            return new GrpcTransport($host, $stubOpts, $channel, $config['interceptors'], $logger);
```

Comment:
Consider keeping this simple
```suggestion
            return new GrpcTransport($host, $stubOpts, $channel, $config['interceptors'], $config['logger'] ?? null);
```